### DISCUSSION TITLE
Can't find  Allowed Google Apps Domains in Settings - General

### DISCUSSION THREAD

**Discussion by eeyoontaek** [getredash/redash#6025] [Q&A]
I am using redash/redash:10.1.0.b50633 image, and I want to connect out company's domain to log in redash.

So I created Google OAuth Client, configured google client id and secret.

Finally I tried to do NO.5 below
<img width="742" alt="스크린샷 2023-05-04 오후 3 37 42" src="https://user-images.githubusercontent.com/118494932/236128531-c2ca5ffa-c769-4352-9461-a4e495c1af42.png">

But I can't find 
<img width="1110" alt="스크린샷 2023-05-04 오후 3 38 40" src="https://user-images.githubusercontent.com/118494932/236128661-1776d958-6a45-4b8f-a5bb-a70890ba1eaf.png">
Allowed Google Apps Domains box.

plz help me

**Comment by kawano-proni** [comment]
Did you restart your redash instance as No.4 says?
